pipelines/release/release.yaml

```diff
@@ -5,7 +5,7 @@ resource_types:
   type: registry-image
   source:
     repository: "govsvc/concourse-github-resource"
-    tag: "v0.0.3"
+    tag: "v1.0.91"
```

Comment:
tag should be `gsp-v1.0.91` ... the old style was likely confusing, clearly this resource has not been bumped since before the gsp prefix was added

can check via: `docker pull govsvc/concourse-github-resource:gsp-v1.0.91`

Reply:
Thanks.